website/Makefile

```diff
@@ -1,7 +1,47 @@
 deps:
-	rsync -av ../ content/ --include='assessments' --include='governance' --include='governance/related-groups' --include='supply-chain-security' --include='*.md' --exclude='*'
+	git submodule update --init --recursive # TODO: the hugo theme doesn't need to live in this directory
+
+	# Move select content from the root level into the website directory.
+	rsync -avv ../ root/ \
+	--include='assessments' --include='assessments/**' \
+	--include='governance' --include='governance/**' \
+	--include='supply-chain-security' --include='supply-chain-security/**' \
+	--include='*.md' --exclude='*'
+
+	# Move over content such as graphics and logos
 	rsync -av '../design/' 'static/design/' --exclude='#*'
-	git submodule update --init --recursive
+
+	# Update all imported markdown files to work as standalone hugo pages (except READMEs, see below)
+	# sed command is configured for the Netlify ubuntu env
+	find root -type f -name '*.md' | while IFS= read -r file; do \
+		base_name=$$(basename "$$file" .md); \
+		if [ "$$file" = "/_index.md" -o "$$base_name" = "README" ]; then \
+			continue; \
+		fi; \
+		text_to_prepend="---\ntitle: \"$$base_name\"\n---\n"; \
+		sed -i "1s/^/$$text_to_prepend/" "$$file"; \
+	done
+
+	# Set up Navbar and Sidebar contents
+	# Add top level dirs to the navbar
+	# Add all subdirectories to the sidebar
+	# If a README is in the dir, copy its contents to the front page for that dir
+	find root -type d -exec sh -c '\
```

Comment:
The only problem I see here currently is that anytime we reference a `README.md` that was translated to an `_index.md` that link is now broken. 

examples from the homepage:
- governance/governance process link
- related-groups link

This may be more widespread throughout the website - IE if you click on the `governance` navbar option and then try `related-groups` you'll encounter an error. yet using the sidebar option for `related groups` works as expected.

Reply:
Will do some experimenting as I have time to think about work-arounds.

Reply:
One work-around - although duplicative - would be to copy both the `README.md` (thus retaining any references to the file) as well as copying the contents of the `README.md` into the `_index.md` file. 

Current `include-markdown` option is breaking links even to direct children. IE from the navbar -> governance -> try to click on any-one of the links - all are broken even though they are present and accessible from the sidebar.

Reply:
I think this is sorted with the latest change. Let me know what you think.

Reply:
agreed! Looks resolved.